### ISSUE TITLE
MongoengineConnectionField paging is SLOW!

### ISSUE THREAD

**Issue by usmcamp0811** [graphql-python/graphene-mongo#125]
I just stood up a MongoDB and started using Graphene-Mongo to serve up the data but am finding `MongoengineConnectionField` queries to be very slow! They work but lets say I want to do:

```
{
    users(first: 10){
         edges{
             node{
                  username
                  full_name
                 }
           }
     }
}
```

That will take probably 5 minutes to execute. If I make my own resolver and just do something like:

```
Users.objects.limit(5)
```

It will be completed in milliseconds. 

Did I miss something in the documentation? Or is this a known issue and I just have to make my own revolvers for everything? 

TIA

**Comment by sostholm** [comment]
I've created a pull request that fixes this issue: https://github.com/sostholm/graphene-mongo/tree/fix-125-MongoengineConnectionField-paging-is-SLOW

**Comment by abawchen** [comment]
@usmcamp0811 : Would you please try with the updated master? I have merged #143 
@sostholm 👍

**Comment by IdrisMiles** [comment]
Was also experiencing this issue, the changes in master did the trick for me, thanks!
